mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/ui/SettingsActivity.kt

```diff
@@ -24,39 +24,46 @@ class SettingsActivity : BaseActivity(), SettingsView {
     @Inject
     var mPresenter: SettingsPresenter? = null
     var mSettingsPresenter: SettingsContract.SettingsPresenter? = null
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_settings)
-        ButterKnife.bind(this)
-        showColoredBackButton(Constants.BLACK_BACK_BUTTON)
-        setToolbarTitle(Constants.SETTINGS)
-        mPresenter!!.attachView(this)
+        setContent {
+            MifosTheme {
+                SettingsScreen(
+                    backPress = { backToHome() },
+                    disable = { onDisableAccountClicked() },
+                    logout = { onLogoutClicked() }
+                )
+            }
+        }
     }
 
-    @OnClick(R.id.btn_logout)
-    fun onLogoutClicked() {
+    private fun onLogoutClicked() {
         val builder = AlertDialog.Builder(this, R.style.AppTheme_Dialog)
```

Comment:
Convert this as well in jetpack compose. First add generic implementation of compose dialog in `core:designsystem` and generic method to directly access the dialog and minimal parameters expose in `core:ui` module and use here.